src/python/pants/backend/python/tasks/select_interpreter.py

```diff
@@ -41,6 +41,8 @@ def compute_fingerprint(self, python_target):
 class SelectInterpreter(Task):
   """Select an Python interpreter that matches the constraints of all targets in the working set."""
 
+  options_scope = 'select-python-interpreter'
```

Comment:
What's this for? `SelectInterpreter` exports no options.

Reply:
If you try to use a task in a test, it will error out if an `options_scope` is not provided (from `test_base.py`):
```python
scope = task_type.options_scope
      if scope is None:
        raise TaskError('You must set a scope on your task type before using it in tests.')
```

Reply:
You want:
https://github.com/twitter/pants/blob/edaf90fdfc5eff087f46971adabce27a22be8bf9/tests/python/pants_test/task_test_base.py#L90

Grepping for that will find several uses.

Reply:
Wow, thanks a lot. I may have snuck in some improper code (setting `options_scope`) in other tests that I will have to fix at some point now (I believe the ctypes unit tests).

Reply:
Fixed as of 79dd8a8c648d5888b42630a81f6f0f32cfffcdef!